wiki/Tournaments/CFC/2018/en.md

```diff
@@ -231,8 +231,8 @@ The Catch French Cup 2018 is run by various French community members by distribu
 ### Match Rules
```

Comment:
I think this is more into

>Match Instructions

rather than Match Rules

Reply:
fair enough, changed